ElementX/Sources/Screens/RoomMemberDetailsScreen/View/RoomMemberDetailsScreen.swift

```diff
@@ -29,7 +29,7 @@ struct RoomMemberDetailsScreen: View {
         }
         .elementFormStyle()
         .alert(item: $context.ignoreUserAlert, actions: blockUserAlertActions, message: blockUserAlertMessage)
-        .errorAlert(item: $context.errorAlert)
+        alert(item: $context.alertInfo)
```

Comment:
Not 100% sure how the unit tests pass in this case unless I'm being very blind and missing something?
```suggestion
        .alert(item: $context.alertInfo)
```